src/helpers/constants/raw.queries.js

```diff
@@ -0,0 +1,9 @@
+const ratingsWithUsers = slug => `SELECT rating, 
+ "Users"."username", "Users"."email", "Users"."firstName", "Users"."lastName", "Users"."profileImage"
```

Comment:
can you use aliases to reduce this SQL query length ? 

https://www.tutorialspoint.com/postgresql/postgresql_alias_syntax.htm

Reply:
Where for e.g ?

Reply:
To avoid repeating users.columName

This is just a suggestion. Sorry I didn't see you that why I put it there

Reply:
Eg: select u.profileImage from Users as u where ...

It will allow to not repeat table's name

This is just a sugestion, I put it because I didn't see you

Reply:
Oh. Cool.

Reply:
What about destructuring the object user  and get.don't you think it will be easier and reduce repetition